src/playcanvas-anim.js

```diff
@@ -412,16 +412,25 @@ Object.assign(window, function () {
      * @param {pc.Vec3} vec3Scale
      * @param {object} output
      */
-    AnimationTarget.constructTargetNodes = function (root, vec3Scale, output) {
+    AnimationTarget.constructTargetNodes = function (root, vec3Scale, output, basePath) {
         if (!root)
             return;
 
+        if (!basePath) {
+            var currNode = root;
+            while(currNode.constructor !== pc.Entity) {
+                currNode = currNode.parent;
+            }
+            basePath = currNode.path + '/' + currNode.children[0].name + '/';
+        }
+        var path = rootTargetNode.targetNode.path.replace(basePath, '');
```

Comment:
This appears to be using rootTargetNode which defined a few lines down?

Reply:
Good catch, thanks! That was introduced in a last minute refactor.